regionGrowing.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from volumeData import VolumeData
+from skimage.filters import threshold_yen
+from skimage.measure import label, regionprops
+from skimage.morphology import closing, square
+
+def srednia(arr, thresh):
+
+    # print('Tab: ', arr)
+    liczbaElementow = np.count_nonzero(arr >= thresh)
+    if liczbaElementow == 0:    # Nie dzielimy przez 0
+        return None
+
+    idxs = np.nonzero(arr >= thresh)
+    suma = sum(arr[idxs])
+    # print(f'Suma: {suma}, Liczba elementów: {liczbaElementow}')
+    return suma / liczbaElementow
+
+def sigma(arr, thresh):
+    """Odchylenie standardowe liczb powyżej progu"""
+    temp = arr - srednia(arr, thresh)
+    var = srednia((arr - srednia(arr,thresh))**2, thresh)
+    if var is None:     # Tylko wtedy gdy mamy w macierzy jeden element
+        return 0
+    else:
+        return np.sqrt(var)
+
+def pointsInRadius(sPoint, radius, dims=2):
+    points = []
```

Comment:
Jeśli szukamy punktów w okręgu to tu jest fajna implementacja:

https://stackoverflow.com/questions/44865023/circular-masking-an-image-in-python-using-numpy-arrays

powinna się dobrze przenieść na 3 wymiary. Dostaniesz maskę którą można użyć do MaskedArray